backend/games/word_guesser/WordGuesser.py

```diff
@@ -0,0 +1,112 @@
+from random import randint
+from typing import Dict, List
+
+MAX_GUESSES = 6
+
+# initialize valid guesses and possible solutions
+SOLUTIONS: str
+VALID_WORDS: str
+
+solutions_file = open("backend/games/word_guesser/WordGuesser_Solutions.txt")
```

Comment:
Should we have exception handling here if this fails? Then the game won't be able to start